radicle-surf/src/diff.rs

```diff
@@ -296,10 +297,39 @@ pub enum DiffContent {
 impl DiffContent {
     pub fn eof(&self) -> Option<EofNewLine> {
         match self {
-            Self::Plain { hunks: _, eof } => Some(eof.clone()),
+            Self::Plain {
+                hunks: _,
+                stats: _,
+                eof,
+            } => Some(eof.clone()),
             _ => None,
         }
     }
+    pub fn stats(&self) -> FileStats {
```

Comment:
```suggestion

    pub fn stats(&self) -> FileStats {
```
New line please :pray: